Packs/CommonTypes/IndicatorFields/incidentfield-description.json

```diff
@@ -14,10 +14,10 @@
     "version": -1,
     "unmapped": false,
     "hidden": false,
-    "type": "longText",
+    "type": "markdown",
     "breachScript": "",
     "editForm": true,
-    "description": "",
+    "description": "re-install CommonTypes if the type is not markdown",
```

Comment:
Pretty sure this wont show up for the customer as it wont actually update this file, or did you check it?

Reply:
seems like it doesn't work, you are right, removing it